### ISSUE TITLE
IPFS-desktop licence appears to be proprietary

### ISSUE THREAD

**Issue by yeehi** [ipfs-shipyard/ipfs-desktop#1712]
- OS: Ubuntu 20.04.1
 - Version of IPFS Desktop [v2.11.4]

**Describe the bug**
The .deb file informs us that the licence is proprietary, whereas in fact ipfs-desktop is released under the permissive MIT licence.

**To Reproduce**
1. Go to 'https://github.com/ipfs-shipyard/ipfs-desktop#install'
2. Download ipfs-desktop for GNU+Linux .deb file 
3. Open the file with debi or package manager and it states it is a proprietary file

**Expected behavior**
The file should indicate its true licence, MIT, which is not proprietary.

**Screenshots**

**Additional context**
![Proprietary Licence error](https://aws1.discourse-cdn.com/standard17/uploads/ipfs1/original/2X/d/d10521c611b3e6637eb93a97e4d20c40a2dc8d50.png)

https://discuss.ipfs.io/t/bug-ipfs-desktop-deb-marked-as-proprietary-software-licence/9297

**Comment by jessicaschilling** [comment]
Thanks, @yeehi -- we'll discuss this in more detail in our next weekly triage.

**Comment by jessicaschilling** [comment]
This may be an issue of dual licensing (Apache and MIT) not being recognized. @Gozala, can you please investigate?

**Comment by yeehi** [comment]
According to @b:

```
".deb is the Debian format, which Ubuntu happens to use. Ubuntu also has its own software center app. That app also acts as a .deb installer.

The apps downloaded from the software centre natively through their centralised service don’t seem to be .debs, they have extra metadata that Debian does not support (?).

… And license information is one of the extra metadata fields the Ubuntu software centre can’t read from deb files, but it can from its own native packages from its own centralised server.

This means any .deb file that you might be able to install will always say ‘proprietary’ because that is what the ubuntu software centre defaults to when it cannot read any licence information from the package. At least this is my best guess.

Similarly, downloaded .deb files cannot have screenshots and a bunch of other stuff that USC native packages can have, so it seems license information is not the only thing Ubuntu only reads from its own Ubuntu-hosted packages, and not from any other source.

Basically, not much we can do - the license info is there, you should just ignore it since it’s never going to be reliable due to very few packages USC can read it from."
```

https://meta.getaether.net/t/aether-flagged-as-proprietary-when-installing-deb/480/3?u=highlandgnu

**Comment by lidel** [comment]
Thank you @yeehi, sounds like this is not something we can fix in IPFS Desktop,  this needs to be tackled upstream.
